services/worker-manager/test/api_test.js

```diff
@@ -208,6 +208,29 @@ helper.secrets.mockSuite(testing.suiteName(), ['taskcluster', 'azure'], function
     throw new Error('delete of non-existent workertype succeeded');
   });
 
+  test('get workertypes', async function() {
```

Comment:
Should we also have a test with >1 workertypes?

Reply:
I was thinking about this. I decided against because then there probably will be the sorting issue - but we sort on the frontend.